user/anyrun/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "anyrun"
+pkgver = "0.0.0"
```

Comment:
```suggestion
pkgver = "0_git20240716"
```

Reply:
Oh! So thats how you do it, thank you very much!